QUEST_LV_0200.tsv

```diff
@@ -1063,32 +1063,32 @@ QUEST_LV_0200_20150317_001062	I am relieved for now.{nl}The rest will be okay wh
-QUEST_LV_0200_20150317_001074	You mean the purification before?{nl}Unlike the other Revelators who were lured to the vicious energy, I purifed you in the name of the Goddess.{nl}
-QUEST_LV_0200_20150317_001075	The Goddesses already blessed the villagers, but they don't know about it well.{nl}The Goddesses don't want chaos.
-QUEST_LV_0200_20150317_001076	It is good to hear that you feel responsible for it.{nl}
-QUEST_LV_0200_20150317_001077	We have something to gain from purifying those demons.{nl}Please obtain the branches of bee trees that are endowed with the power of the Goddess near the village.
-QUEST_LV_0200_20150317_001078	We should complete it quickly before the monsters rush in again.
-QUEST_LV_0200_20150317_001079	You brought them. Good.{nl}
-QUEST_LV_0200_20150317_001080	I told you a while ago that there are two sources of vicious energy.{nl}Among them, I will make a bead that will restore the seal that is located here.
-QUEST_LV_0200_20150317_001081	First, bring me the ashes of demons to be used as ingredient for the bead.{nl}When you pierce the demon with the branch of the bee tree, it won't be able to resist the divine power and become ashes.{nl}
-QUEST_LV_0200_20150317_001082	But of course, the power of Goddess resides indirectly in it. {nl}So you will have to consume some of the demon's HP before using it.
-QUEST_LV_0200_20150317_001083	Good work. {nl}It's a uncomfortable to use demons as sacrifice.. Now we're in the final step.
-QUEST_LV_0200_20150317_001084	Take this bead. I made it in a hurry so it's not yet complete. {nl}You must fill it with the divine spell of the Goddess.
-QUEST_LV_0200_20150317_001085	Palama Cliff is the nearest place for it so please head there. {nl}The demons will soon be marching in so hurry.
-QUEST_LV_0200_20150317_001086	This will be enough. {nl}Please prepare to restore the seal. It might become a huge fight.
-QUEST_LV_0200_20150317_001087	The weak seal tower is in the near Stula Road. {nl}It bothers me that the demons were calm until now.
-QUEST_LV_0200_20150317_001088	Please hurry. I can feel that the seal is weak. {nl}May you have the Goddess' blessings..
-QUEST_LV_0200_20150317_001089	Are you the Revelator who purified the Seal Tower?
-QUEST_LV_0200_20150317_001090	I have been suppressing the evil force of this place.
-QUEST_LV_0200_20150317_001091	Please restore the Seal Tower in Spring Light Wood.
+QUEST_LV_0200_20150317_001074	$The purification back then?{nl}Unlike the other Revelators who succumbed to evil energy, I purified you in the name of the Goddess.{nl}
+QUEST_LV_0200_20150317_001075	$The Goddesses have already blessed the villagers, they just don't know about it.{nl}The Goddesses don't want chaos.
+QUEST_LV_0200_20150317_001076	$It is good to hear that you feel involved.{nl}
+QUEST_LV_0200_20150317_001077	$We need to purify those demons to obtain something.{nl}Please obtain the branches of the bee trees near the village. They are endowed with the power of Goddess Austeja.
```

Comment:
I think it should be "the Goddess Austeja." Unless I missed some issue where you guys decided not to put the infront of Goddess.

We've put the infront of Goddess in various other places tho, and we also do it with the Master titles.